src/gui/plot/qgselevationprofilecanvas.h

```diff
@@ -0,0 +1,163 @@
+/***************************************************************************
+                          qgselevationprofilecanvas.h
```

Comment:
this file would better live in src/gui/elevation no?

Reply:
Done